src/main/scala/org/constellation/crypto/KeyStoreUtils.scala

```diff
@@ -83,7 +83,7 @@ object KeyStoreUtils {
     alias: String,
     storePassword: Array[Char],
     keyPassword: Array[Char]
-  ): F[KeyStore] =
+  ): F[Either[Throwable, KeyStore]] =
```

Comment:
Why not EitherT?

Reply:
Just a leftover, I accidentally moved EitherT conversion to other place. On it.